sdks/java/io/amazon-web-services2/src/main/java/org/apache/beam/sdk/io/aws2/kinesis/KinesisReader.java

```diff
@@ -39,62 +40,42 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(KinesisReader.class);
 
+  private final Read read;
```

Comment:
Better to name it `spec` as we used to do for other IOs

Reply:
👍